{"text": "create_engine('postgresql://root:root@localhost:5432/ny_taxi')  I get the error \"TypeError: 'module' object is not callable\"\nSolution:\nconn_string = \"postgresql+psycopg://root:root@localhost:5432/ny_taxi\"\nengine = create_engine(conn_string)", "section": "Module 1: Docker and Terraform", "question": "Python - SQLALchemy - TypeError 'module' object is not callable", "course": "data-engineering-zoomcamp", "document_id": "d44d1c77"}